modules/dxp/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/ClickGoalPicker/ClickGoalPicker.es.js

```diff
@@ -113,6 +121,28 @@ function ClickGoalPicker({allowEdit = true, onSelectClickGoalTarget, target}) {
 		dispatch({type: 'activate'});
 	};
 
+	const handleInputBlur = (event) => {
+		const value = event.target.value;
+
+		const target = value && document.getElementById(value);
+
+		if (target) {
+			target.scrollIntoView();
+
+			// Make sure nothing slides under the top nav.
+
+			window.scrollBy(0, -100);
```

Comment:
Magical value should probably at least have a named constant like `TOP_NAV_HEIGHT` or something; ideally we'd have this defined somewhere already that you could get a hold of it.

Reply:
Ok, sure, I'll create a constant :)